From 19b9c8dc7f45c366acc866426d1a84da5a6a44bf Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Thu, 24 Jun 2021 10:58:08 +0200
Subject: [PATCH] Drop duplicate `ComputedEffectTiming` dfn

The dictionary is re-defined in Level 2 with an additional property and
different property types. Patch is only needed because Level 2 is a delta spec.
It will likely need to be kept around for as long as that remains the case.
---
 ed/idl/web-animations.idl | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/ed/idl/web-animations.idl b/ed/idl/web-animations.idl
index 6e973d195..a9675b1d9 100644
--- a/ed/idl/web-animations.idl
+++ b/ed/idl/web-animations.idl
@@ -85,14 +85,6 @@ enum FillMode { "none", "forwards", "backwards", "both", "auto" };
 
 enum PlaybackDirection { "normal", "reverse", "alternate", "alternate-reverse" };
 
-dictionary ComputedEffectTiming : EffectTiming {
-    unrestricted double  endTime;
-    unrestricted double  activeDuration;
-    double?              localTime;
-    double?              progress;
-    unrestricted double? currentIteration;
-};
-
 [Exposed=Window]
 interface KeyframeEffect : AnimationEffect {
     constructor(Element? target,
-- 
2.32.0.windows.1

